spectrum-offchain-lm/src/main.rs

```diff
@@ -186,7 +186,7 @@ async fn main() {
         schedules,
         bundles,
         &node,
-        20,
+        10, // Note: setting this higher could lead to rejection of compound orders by Ergo Node.
```

Comment:
Very important to cap the bundles to 10 per compound TX. This was tested empirically.